src/content/contributing/release-process/_index.en.md

```diff
@@ -4,145 +4,225 @@ date: 2020-03-18T16:03:26+01:00
-It is assumed that you are familiar with the submariner project and the various repositories.
+This section describes how to create a Submariner product release. It is assumed that you are familiar with the various
+Submariner projects and their repositories and are familiar with Git and GitHub.
 
-## Step 1: Create a Submariner Release
+## Project dependencies
 
-Assuming that you have an existing submariner git directory, the following steps create a release named "Globalnet Overlapping IP support RC0" with version v0.2.0-rc0 based on the master branch.
+The Submariner projects have a dependency hierarchy with respect to their Go libraries and Docker images.
+Therefore the releases for each individual project must be created in a specific order.
 
-```bash
-cd submariner
-git stash
-git remote add upstream ssh://git@github.com/submariner-io/submariner
-git fetch -a -v -t upstream
-git checkout remotes/upstream/master -B master
-git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
-git push upstream v0.2.0-rc0
-```
+The Go dependency hierarchy flow is as follows:
 
-A tagged release should appear [here](https://github.com/submariner-io/submariner/tags).
+`shipyard` <- `admiral` <- `[submariner, lighthouse]` <- `submariner-operator`
 
-> https://github.com/submariner-io/submariner/tags
+Note that the `submariner` and `lighthouse` projects are siblings and thus do not depend on one another. Also the
+`submariner-operator` components expect that `lighthouse` and `submariner` are aligned on the same exact version.
 
-A build for v0.2.0-rc0 should start and appear under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/submariner/branches).
+The Docker image dependency hierarchy flow is as follows:
 
-> https://travis-ci.com/github/submariner-io/submariner/branches
+`subctl binary` <- `shipyard dapper base image` <- `[admiral, submariner, lighthouse, submariner-operator]`
 
-Verify that the build successfully completes as indicated by a green checkmark at the right. At this point the images tagged with 0.2.0-rc0 will be available [here](https://quay.io/repository/submariner/submariner?tab=tags).
+The Dapper base image that is provided by `shipyard` for building and E2E tests in all of the other projects pulls in the
+`subctl` binary.
 
-> https://quay.io/repository/submariner/submariner?tab=tags
+## Release versions
+
+The `vx.x.x` version format is used for the git projects while `x.x.x` is used for Docker images.
+
+The typical workflow is to first create release candidate(s) for testing before creating the final release. The suggested
+naming convention is to append `-rcN` to the final version, for example `v0.5.0-rc0`, `v0.5.0-rc1` etc.
+
+Sometimes you may want to create a specific project release for testing prior to creating a release candidate. In this
+case, the suggested naming convention is to append `-preN`.
+
+## Create Submariner product release
```

Comment:
```suggestion
## Create a submariner-io github org product release
```

Suggestion to clarify further based on mike comments

Reply:
This section is meant as an intro to creating a full Submariner **product** release, which encompasses all the github projects. The intro segues into the individual steps as subsections. I could add a subsection for  "Create a GitHub 
 project release" instead of inlining it.

Reply:
Ah ok, this is ## and the other is ###

Reply:
ok, my suggestions should be ignored then